src/instruments/src/SD/Pages/Apu/Apu.tsx

```diff
@@ -0,0 +1,514 @@
+                                value={102}
+                                radius={50}
+                                startAngle={240}
+                                multiplierOuter={1.2}
+                                endAngle={60}
+                                className="NoFill AmberHeavy"
+                                outer
+                            />
+                            {apuN.isNormalOperation()
+                        && (
+                            <GaugeMarkerComponent
+                                x={0}
+                                y={50}
+                                min={0}
+                                radius={50}
+                                max={120}
+                                startAngle={240}
+                                endAngle={60}
+                                value={Number.parseFloat(apuN.value.toFixed())}
+                                className={`Line ${apuNIndicationColor}`}
+                                indicator
+                            />
+                        )}
+                        </GaugeComponent>
+                    </GaugeComponent>
+                </SvgGroup>
+
+                <SvgGroup x={80} y={13}>
+                    <text x={0} y={0} className="White Center FontNormal">N</text>
+                    <text x={0} y={30} className="Cyan Center FontNormal">%</text>
+                </SvgGroup>
+
+                <text
+                    x={10}
+                    y={80}
+                    className={`FontLarger Left ${apuN.isNormalOperation() ? apuNIndicationColor : 'Amber'}`}
+                >
+                    {apuN.isNormalOperation() ? apuN.value.toFixed() : 'XX'}
+                </text>
+            </SvgGroup>
+        </>
+    );
+};
+
+const EgtGauge = ({ x, y } : ComponentPositionProps) => {
+    const apuEgt = useArinc429Var('L:A32NX_APU_EGT', 100);
+    const displayedEgtValue = Math.round(apuEgt.value / 5) * 5; // APU Exhaust Gas Temperature is shown in steps of five.
+
+    const apuEgtCaution = useArinc429Var('L:A32NX_APU_EGT_CAUTION', 500);
+    const apuEgtWarning = useArinc429Var('L:A32NX_APU_EGT_WARNING', 500);
+
+    const [apuEgtIndicationColor, setApuEgtIndicationColor] = useState('Green');
+
+    const redLineShown = apuEgtCaution.isNormalOperation() && apuEgtWarning.isNormalOperation();
+
+    useEffect(() => {
+        if (apuEgt.value <= apuEgtCaution.value) {
+            setApuEgtIndicationColor('Green');
+        }
+        if (apuEgt.value > apuEgtCaution.value && apuEgt.value <= apuEgtWarning.value) {
+            setApuEgtIndicationColor('Amber');
+        }
+        if (apuEgt.value > apuEgtWarning.value) {
+            setApuEgtIndicationColor('Red');
+        }
+        // FBW-31-05
+    }, [apuEgt, apuEgtCaution, apuEgtWarning]);
+
+    let egtValueStyle;
+
+    if (!apuEgt.isNormalOperation()) {
+        egtValueStyle = 'Amber';
+    } else if (apuEgtIndicationColor === 'Pulse') {
```

Comment:
**thought**: I don't see where `apuEgtIndicationColor` is ever set to pulse.

Reply:
Hmm I think we had a chat about this a while back. Because we don’t have the ECB currently modeled, which is what signals the pulsing to occur, I think we agreed to omit it for now so it seems like this line of code is a vestige of that.

Reply:
Probably removing this piece of code for now is best. Many things in the ECB are modelled, but maybe not this particular thing.

Reply:
Maybe it's best to remove this remnant for now?

Reply:
Yeah I agree.